x/bridge/keeper/params_test.go

```diff
@@ -0,0 +1,104 @@
+package keeper_test
+
+import (
+	"cosmossdk.io/math"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/gogoproto/proto"
+
+	"github.com/osmosis-labs/osmosis/v23/x/bridge/types"
+)
+
+func (s *KeeperTestSuite) TestUpdateParams() {
```

Comment:
Lets set this up as a table driven test like the other ones, there is repetitive code here